### PR TITLE
Web3 integration

### PR DESCRIPTION
**Pull request by TForge1**
Added Web3 Functions, Adjusted entity card to produce an entropy value from on-chain mapping.

## Summary by CodeRabbit

- **Documentation**
  - Enhanced the `README.md` description to better highlight TraitForge's features and technologies.
- **Refactor**
  - Improved code readability and maintainability across various components without affecting functionality.
  - Adjusted JSX structure and CSS for better presentation in UI components.
  - Updated utility functions and configurations for better application performance.
- **New Features**
  - Updated the entity attribute calculation process to enhance gameplay experience.
  - Refined the Slider component for upcoming mints and entity prices.
- **Style**
  - Comprehensive updates to CSS and SCSS files to improve visual presentation and consistency.
- **Tests**
  - Refactored test files for consistency and readability without changing the test logic.

### CODE REVIEW

**Comment by TForge1** [comment]
i have not done any tailwind or CSS changes, if that is something you can do stefan?

File: client/components-v1/ForgingModal.jsx [hunk 1]
```diff
@@ -1,6 +1,9 @@
 import React, { useState } from 'react';
 import '../styles/TradingForgingModal.css';
-import { useWeb3ModalProvider, useWeb3ModalAccount } from '@web3modal/ethers5/react';
+import {
+  useWeb3ModalProvider,
+  useWeb3ModalAccount,
+} from '@web3modal/ethers5/react';
 import { ethers } from 'ethers';
 import forgeContractAbi from '../artifacts/contracts/EntityMerging.sol/EntityMerging.json';
```
Comment (sstefdev) [inline client/components-v1/ForgingModal.jsx:8]: please add this into the config file

File: client/components-v1/ForgingModal.jsx [hunk 2]
```diff
@@ -61,12 +71,14 @@ const ForgingModal = ({ open, onClose, onSave }) => {
   if (!open) return null;
 
   return (
-    <div className='overlay'>
-      <div className='modalContainer'>
-        <button className='closeBtn' onClick={onClose}>x</button>
+    <div className="overlay">
```
Comment (sstefdev) [inline client/components-v1/ForgingModal.jsx:74]: we are using modulated SCSS, these classes wont work properly...

Comment (TForge1) [reply client/components-v1/ForgingModal.jsx:74]: these were pulled out of old components, my fault.

Comment (TForge1) [reply client/components-v1/ForgingModal.jsx:74]: actualy ignore all of the v1s because they were added to PR because of prettier. they arent used, ignore

**Pull request closed without merging**